SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -56,6 +57,22 @@ struct ConfigureConnection
 		Params.network.modular_kcp.upstream_heartbeat = &HeartbeatParams;
 #endif
 
+		if (*Config.WorkerType == SpatialConstants::DefaultServerWorkerType.ToString() && GetDefault<USpatialGDKSettings>()->bUseSecureServerConnection)
```

Comment:
Unless I'm mistaken, it was `*Config.WorkerType != SpatialConstants::DefaultServerWorkerType.ToString()` before?

Reply:
We've changed it so you can enable secure connection both servers and clients. Hence this additional check.

Reply:
Cool 👍